modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json

```
@@ -308,17 +308,23 @@
 									{
 										"definition": {
 											"fragment": {
-												"key": "BASIC_COMPONENT-html"
+												"key": "get-app-button",
+												"siteKey": "[$GROUP_KEY$]"
 											},
 											"fragmentConfig": {
+												"productId": {
+													"href": ""
+												}
 											},
 											"fragmentFields": [
 												{
-													"id": "element-html",
+													"id": "productID",
```

Please, change the id name.

```suggestion
													"id": "product-id",
```